src/chapters/thymeleaf-views/exercises.rst

```diff
@@ -0,0 +1,15 @@
+Exercises: Thymeleaf
```

Comment:
What you have here seems fine, but maybe a little short.

Perhaps we could have the students modify the form to collect a description as well as an event name?  This would involve refactoring the code to use a HashMap instead of an ArrayList, however.

Reply:
Added a bonus mission!